includes/shortcodes/class-sensei-legacy-shortcodes.php

```diff
@@ -235,6 +235,7 @@ public static function user_courses( $atts, $content = null ) {
 
 		if ( is_user_logged_in() ) {
 
+			wp_enqueue_script( Sensei()->token . '-tabs' );
```

Comment:
This is called outside the `wp_enqueue_scripts` scripts hook, which is not recommended. Calling it here shouldn't cause any problems - it's not called before `init`, it's only loading JS, in the footer, without any localization. Since it's a deprecated hook anyway, the whole thing can also be removed easier in the future this way.